source/Makefile

```diff
@@ -126,14 +123,11 @@ OUTPUT_DIR=Objects/$(model)
 OPTIM=-Os -flto -finline-small-functions -findirect-inlining -fdiagnostics-color -ffunction-sections -fdata-sections
 
 # global defines ---------------------------------------------------------------
-GLOBAL_DEFINES += $(DEV_GLOBAL_DEFS) -D USE_RTOS_SYSTICK -D LANG_$(lang) -D LANG -D MODEL_$(model) -D VECT_TAB_OFFSET=$(bootldr_size)U
+GLOBAL_DEFINES += $(DEV_GLOBAL_DEFS) -D USE_RTOS_SYSTICK -D MODEL_$(model) -D VECT_TAB_OFFSET=$(bootldr_size)U
 
 # Without debug code
 DEBUG=
 ifdef swd_enable
-	GLOBAL_DEFINES += -DSWD_ENABLE

```

Comment:
You appear to have removed the SWD_ENABLE flag?

Reply:
Indeed, will readd.